Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -69,9 +69,19 @@
 #include "DolphinWX/resources/Platform_Wii.xpm"
 #include "DolphinWX/resources/rating_gamelist.h"
 
-size_t CGameListCtrl::m_currentItem = 0;
-size_t CGameListCtrl::m_numberItem = 0;
-std::string CGameListCtrl::m_currentFilename;
+struct CompressionProgress final
+{
+public:
+	CompressionProgress(int items_done, int items_total, const std::string& current_filename, wxProgressDialog& dialog)
+		: items_done(items_done), items_total(items_total), current_filename(current_filename), dialog(dialog)
+	{ }
+
+	int items_done;
```

Comment:
You can just add a `_` at the end of the constructor items, if you prefer non-`m_` notation for these.